simulator/src/simulator/simulation-node.ts

```diff
@@ -509,6 +510,58 @@ export class SimulationNode {
     this.onLog.clear()
     this.onError.clear()
   }
+
+  /**
+   * Executes a cli command command via `child_process.exec()` synchronously and returns the stdout and stderr.
+   * If you need the command to be asynchronous, use `executeCliCommandAsync` instead.
+   *
+   * @param args The ironfish cli arguments to execute
+   * @param callback The callback to execute when the command is complete
+   */
+  executeCliCommand(
+    args: string[],
+    callback?: (err: ExecException | null, stdout: string, stderr: string) => void,
```

Comment:
Could we standardize the callbacks here? It would be nice if this function took similar arguments to addNode like onLog and onError, onExit. If we want to we could standardize those callbacks into a type. Don't think it's necessary with just 2 uses yet though

Reply:
addr'd